freshmaker/lightblue.py

```diff
@@ -42,6 +42,25 @@
 import koji
 
 
+class ImageGroup:
+    def __init__(self, image, lightblue):
+        parsed_nvr = koji.parse_NVR(image.nvr)
+        repositories = image.get_registry_repositories(lightblue)
+        self.name = parsed_nvr["name"]
+        self.version = parsed_nvr["version"]
+        self.repos = {x["repository"] for x in repositories}
+
+    def __eq__(self, other):
```

Comment:
Since you're choosing to go with the helper class, you can just do some minor changes to the helper class as I suggested before, check this gist (https://gist.github.com/qixiang/89df6f26db5b479c3fdd168c77fff660) for reference, I didn't test it (also not updated comments or function docs), in theory, it should work.

Reply:
gotcha, changes have been made!